models/topics.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection.js");
+const fs = require("fs/promises");
+
+exports.selectAllTopics = () => {
+  return db.query("SELECT * FROM topics").then((res) => {
+    return res.rows;
+  });
+};
+
+exports.selectAllEndpoints = () => {
```

Comment:
We don't actually need a model here, we can just require in the endpoints.json into the controllers file, then send it as a response from there 🙂

Reply:
I ill refactor all now, thank you